src/system/config-wfrp4e.js

```diff
@@ -1891,7 +1892,7 @@ WFRP4E.PrepareSystemItems = function() {
                 scriptData: [
                     {
                         trigger: "manual",
-                        label: "@effect.name",
+                        label: game.i18n.localize("WFRP4E.ConditionName.Bleeding"),
```

Comment:
I'm not sure why this is necessary if the effect name itself is localized?

Reply:
i noticed in some cases, that the effect itself was just "@effect.name". I guess it doesn't matter that much, since those are mostly hidden but i thought about keeping it consistent. 

If that's a problem, i will revert